packages/owl-bot/src/owl-bot.ts

```diff
@@ -101,16 +109,18 @@ export = (privateKey: string | undefined, app: Probot, db?: Db) => {
       },
       context.octokit
     );
-
-    await context.octokit.issues.removeLabel({
-      name: OWLBOT_RUN_LABEL,
-      issue_number: prNumber,
-      owner,
-      repo,
-    });
+    try {
+      await context.octokit.issues.removeLabel({
+        name: OWLBOT_RUN_LABEL,
+        issue_number: prNumber,
+        owner,
+        repo,
+      });
+    } catch (err) {
+      logger.error(`${err.message} head = ${head} pr = ${prNumber}`);
```

Comment:
Once we fail once, we've already removed the `OWLBOT_RUN_LABEL` and so get errors on subsequent runs, I'll loosen this error as soon as I confirm the error code that's thrown.